src/installer/corehost/cli/apphost/static/CMakeLists.txt

```diff
@@ -120,21 +121,89 @@ if(CLR_CMAKE_TARGET_NETBSD)
     )
 endif(CLR_CMAKE_TARGET_NETBSD)
 
-if(NOT CLR_CMAKE_TARGET_WIN32)
-    set(DEF_SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/singlefilehost_unixexports.src)
-    set(EXPORTS_FILE ${CMAKE_CURRENT_BINARY_DIR}/singlefilehost.exports)
-    generate_exports_file(${DEF_SOURCES} ${EXPORTS_FILE})
+# Path like: artifacts/bin/native/net5.0-Linux-Release-arm/
+set(NATIVE_LIBS_LOCATION "$ENV{__NativeLibsArtifacts}")
```

Comment:
Can we switch this to being a CMake variable passed in instead of an environment variable?

Reply:
I would need an example for this - to see how it is done.
`__NativeLibsArtifacts` is set up by `build.sh/build.bat` . There are many examples there for passing arguments via environment, so I used similar approach for `__NativeLibsArtifacts`.
I do not have a lot of preferences though.

Reply:
I've added a suggestion for how to change the variable to be passed in.